.storybook/main.ts

```diff
@@ -25,16 +26,47 @@ module.exports = {
 
     const customStorybookConfig = defineConfig({
       base,
+      define: {
+        'process.browser': 'true',
+      },
       plugins: [
+        // TODO: The following will fix SynapseFormWrapper in development mode but will break the storybook static build
+        // Need to figure out why the polyfill breaks the build
+        // nodePolyfills({
+        //   include: 'node_modules/@apidevtools/json-schema-ref-parser/**/*.js',
+        // }),
         svgr(),
         viteExternalsPlugin({
           react: 'React',
           'react-dom': 'ReactDOM',
           'sql-parser': 'SQLParser',
-          '@sage-bionetworks/rjsf-core': 'JSONSchemaForm',
-          '@rjsf/core': 'JSONSchemaForm',
```

Comment:
Fixed in `@storybook/builder-vite` v0.2.x, so we can now include RJSF in the bundle